bootstrap_cfn/fab_tasks.py

```diff
@@ -1074,8 +1094,12 @@ def get_all_elbs(f=None):
     returns True, or everything.
     """
     cfn_config = get_config()
-    elbs = [x.get('name') for x in cfn_config.data.get('elb', {}) if x.get('scheme') in ['internet-facing', 'internal']]
-    return filter(f, elbs) if f else elbs
+    elbs = cfn_config.data.get('elb', {})
```

Comment:
if you don't need `f` skip the parameter. If you don't need to filter remove the parameter